src/components/index.ts

```diff
@@ -1,12 +1,17 @@
-export * from "./address-text";
-export * from "./copy-to-clipboard";
-export * from "./data-field";
-export * from "./form";
-export * from "./icons";
-export * from "./modal";
-export * from "./nav";
-export * from "./nav-item";
-export * from "./option-card";
-export * from "./selectors";
-export * from "./steps";
-export * from "./txn-expire-text";
+// Export all ChakraUI components
+export * from "@chakra-ui/react"
```

Comment:
Re-exporting the whole ChakraUI package here. In another PR I'll go through and make sure we're importing components from this file instead of from Chakra directly (to allow us to override as we currently do with Modals).